.github/workflows/ci.yml

```diff
@@ -14,21 +14,21 @@ concurrency:
 jobs:
   change-detection:
     name: 🔍 Change
-    uses: cda-tum/mqt-workflows/.github/workflows/reusable-change-detection.yml@v1.0.0
+    uses: cda-tum/mqt-workflows/.github/workflows/reusable-change-detection.yml@v1.0.1
 
   python-tests:
     name: 🐍 Test
     needs: change-detection
     if: fromJSON(needs.change-detection.outputs.run-python-tests)
-    uses: cda-tum/mqt-workflows/.github/workflows/reusable-python-ci.yml@v1.0.0
+    uses: cda-tum/mqt-workflows/.github/workflows/reusable-python-ci.yml@v1.0.1
     secrets:
       token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
You need to remove these two lines as they are no longer necessary. Coverage uploads no longer need a token

Reply:
Done.